scss/adminLessonCard.module.scss

```diff
@@ -0,0 +1,40 @@
+@use './colors.module';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 30px 53px 30px 53px;
```

Comment:
A shorthand would be `padding: 30px 53px`

Reply:
This should be updated